documentation/profile-migration-guide.md

```diff
@@ -0,0 +1,33 @@
+# Guide for migrating from `profiles/**`
+
+This document is intended for users that are familiar with the previous version of the Azure SDK For Go profiles package (`github.com/Azure/azure-sdk-for-go/profiles/**`) and wish to migrate their application to the next version of Azure SDK for Go.
+
+## Table of contents
+
+- [Prerequisites](#prerequisites)
+- [Package mapping](#package-mapping)
+- [API changes](#API-changes)
+
+## Prerequisites
+
+- Go 1.18
+
+## Package mapping
+
+There are several sub-packages under pervious version of the Azure SDK For Go profiles package (`profiles/**`): `2017-03-09`, `2018-03-01`, `2019-03-01`, `2020-09-01`, `latest`, `preview`.
+
+For those who are using `latest` or `preview`, you need to migrate to our latest modules under `github.com/Azure/azure-sdk-for-go/sdk`. The latest SDK only provides one API version in one module for each RP. If you are using several RP's sub-packages, you need to migrate to several modules. You can search in [pkg.go.dev](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go) to find all the stable and preview version of all RPs.
```

Comment:
”The latest SDK only provides one API version in one module for each RP. If you are using several RP's sub-packages, you need to migrate to several modules. “ a little difficult to understand this.  The proposal according to my understanding:

... However, the modules of different resource providers are not packaged all in one.  Only the module(s) of the same resource provider is(are) packaged together. To migrate to the latest modules, you need to find all packages that contain required modules.  You can search in pkg.go.dev to find those resource providers and new releases.

Reply:
Changed some expressions.